iina/FileGroup.swift

```diff
@@ -17,7 +17,6 @@ class FileInfo: Hashable {
   var filename: String
   var ext: String
   var nameInSeries: String?
-  var characters: [Character]
```

Comment:
I don't see any obvious performance issue at this point. The amount of data isn't large enough, after all.